test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.0.2"
```

Comment:
If we are not buildign CLI for any new platforms than what is supported in 1.0.0, we dont need to change the version (you will still see break with the version update, all the same) but specify the RTM version of CLI as I mentioned in a separate comment.